src/main/kotlin/no/nav/k9/jobber/Statistikkjobb.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.k9.jobber
+
+import no.nav.k9.domene.repository.StatistikkRepository
+import org.slf4j.LoggerFactory
+import java.time.temporal.ChronoUnit
+import java.util.concurrent.TimeUnit
+import kotlin.concurrent.fixedRateTimer
+
+class Statistikkjobb(private val statistikkRepository: StatistikkRepository) {
+
+    private val log = LoggerFactory.getLogger(Statistikkjobb::class.java)
+
+    fun kjørOppgaveOgBehandlingsstatistikk() {
+        fixedRateTimer(
+            name = "statistikk",
+            daemon = true,
+            initialDelay = TimeUnit.SECONDS.toMillis(10),
+            period = TimeUnit.DAYS.toMillis(1)) {
+            `hvor lang tid tar det fra en sak kommer inn til k9 til en saksbehandler tar tak i den`()
+            //`hvor lenge ligger journalføringsoppgaver før de håndteres`()
+            //`hvor lenge ligger saker på vent i k9`()
+        }
+    }
+
+    private fun `hvor lang tid tar det fra en sak kommer inn til k9 til en saksbehandler tar tak i den`() {
+        val tider = mutableListOf<Long>()
+
+        for (reservasjon in statistikkRepository.hentReservasjoner()) {
+            if (statistikkRepository.erOppgavenAktiv(oppgaveId = reservasjon.oppgave)) {
+                val behandlingOpprettet = statistikkRepository.hentBehandlingOpprettet(eksternReferanse = reservasjon.oppgave)
```

Comment:
jeg tror dette kan bli tungt fordi du henter alle reservasjoner (både aktive og ikke aktive) og for hver av dem sjekkes det om oppgave er aktiv fra oppgave tabell. Også sjekkes behandling tabellen. Jeg tror det vil være bedre ytelsesmessig å hente dette ferdig joina fra med SQL.

Reply:
Ja, jeg vurderte det samme. Det spørs kanskje litt hvor mye sjekker som må gjøres på jsonb-kolonner tror jeg. Jeg skal undersøke